outputs.tf

```diff
@@ -87,3 +87,13 @@ output "eks_cluster_ca_certificate" {
   description = "CA certificate"
   value       = "module.eks_cluster.certificate_authority[0].data"
 }
+
+output "cluster_endpoint" {
+  description = "The endpoint of the EKS cluster"
+  value       = module.eks_cluster.eks_cluster_endpoint
```

Comment:
Why we are duplicating the outputs?
If you see there is already an output named 
`eks_cluster_endpoint` and `eks_cluster_version`

Reply:
fixed

Reply:
Do you have any more changes to add? as of now there is no file change that i can see @vigneshnagachalavelavan